examples/standalone/benchmarks/plotting.py

```diff
@@ -19,10 +22,38 @@
     args = parser.parse_args()
 
     # setup what to plot
-    plots = {}
-    plots["mainLoop"] = ["mainloop", "DynCore", "Remapping", "TracerAdvection"]
-    plots["initializationTotal"] = ["initialization", "total"]
-    backends = ["python/gtx86", "python/numpy", "fortran", "python/gtcuda"]
+    fontsize = 10
+    markersize = 4
+    plot_variance = True
+    plots = {
+        "per_timestep": {
+            "timers": ["mainloop", "DynCore", "Remapping", "TracerAdvection"],
+            "title": "Performance history of components",
+            "x_axis_label": "Date of benchmark",
+            "y_axis_label": "Execution time per timestep [s]",
+        },
+        "absolute_time": {
+            "timers": ["total", "initialization"],
+            "title": "Performance history of absolute runtime",
+            "x_axis_label": "Date of benchmark",
+            "y_axis_label": "Execution time [s]",
+        },
+    }
+    linestyles = {
```

Comment:
Thanks for the suggestion. I agree. I made it a dict with keys, so that it is clear that the information refers to a linestyle and to avoid `timer[0]` in the code. This should also ease moving this static data into a readable JSON configuration file at some point.